### PR TITLE
[#63] Implement 'hit status' command

### LINKED ISSUE #63: Implement `hit status` command with nice output
See example of the output here:

* https://stackoverflow.com/a/55202319/2900502

### PR DESCRIPTION
**Pull request by chshersh**
Resolves #63

Here is how the output looks like:

Screenshot from 2019-07-04 22-20-43

### CODE REVIEW

File: src/Hit/Git/Status.hs [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+data PatchType
+    = Added
+    | Copied
+    | Deleted
+    | Modified
+    | Renamed
+    | TypeChanged
+    | Unmerged
+    | Unknown
+    | BrokenPairing
```
Comment (vrom911) [inline src/Hit/Git/Status.hs]: Sometimes it seems like all git types and commands etc/ deserve their own library.

Comment (chshersh) [reply src/Hit/Git/Status.hs]: Yeah, having nice CLI wrapper around `git` would be really nice...

**Comment by chshersh** [comment]
@vrom911 Thanks for your kind words! I agree that we can use it as it is for now and decide how to improve later. I often have better ideas about improvements only after using for a while :slightly_smiling_face:

File: src/Hit/Cli.hs [hunk 2]
```diff
@@ -79,7 +79,7 @@ hitP = subparser
    <> command "sync"    (info (helper <*> syncP)    $ progDesc "Sync local branch with its remote")
    <> command "resolve" (info (helper <*> resolveP) $ progDesc "Switch to master, sync and delete the branch")
    <> command "current" (info (helper <*> currentP) $ progDesc "Show info about current branch and issue (if applicable)")
-   <> command "status"  (info (helper <*> statusP)  $ progDesc "Show current branch and beatiful diff")
+   <> command "status"  (info (helper <*> statusP)  $ progDesc "Show current branch and beautiful diff with HEAD")
```
Comment (vrom911) [inline src/Hit/Cli.hs]: I've noticed that it's not only with HEAD bt with any given commit. HEAD is the default

Comment (chshersh) [reply src/Hit/Cli.hs]: I've updated the text to be more precise

**Pull request merged**